### DISCUSSION TITLE
Guidance on install

### DISCUSSION THREAD

**Discussion by paulcusick3** [keatontaylor/alexa-actions#239] [General]
Where do I go for this?   I have an old version, the setup was very different.... can not find this.

Go to the latest release of the skill and download the zip file called AlexaActionsNoBinary.zip

**Comment by DEADSEC-SECURITY** [comment]
What exactly is the issue? Could you be more specific what part of the wiki should be improved?

**Comment by paulcusick3** [comment]
Sure - Where to find the zip file.  

Go to the latest release of the skill and download the zip file called AlexaActionsNoBinary.zip

I don't see that file here:
https://github.com/keatontaylor/alexa-actions/

In prior installs I just copied the raw files 1 at a time.  I've searched for that file name.  Assuming it is not a zip of the repository.

**Reply by Flutzi** [reply]
Hey! 

I just downloaded the file you see below. (Code > Download Zip)

![Bildschirmfoto 2024-03-07 um 19 08 50](https://github.com/keatontaylor/alexa-actions/assets/160764196/92c04606-9794-40a5-a215-33b3cc8b25dc)

Afterwards, I packed the downloaded file into a zip archive and then uploaded it to the Developer Console.

**Comment by paulcusick3** [comment]
Ok - I see it is just a zip of the git... will try

**Comment by DEADSEC-SECURITY** [comment]
You have to go to releases not the repo files.
https://github.com/keatontaylor/alexa-actions/releases/tag/v0.12.2
